guides/common/modules/con_creating-and-managing-roles.adoc

```diff
@@ -1,7 +1,33 @@
 [id="Creating_and_Managing_Roles_{context}"]
+ifeval::["{context}" == "admin"]
 = Creating and managing roles
+endif::[]
+ifeval::["{context}" == "planning"]
+= Defining role-based access control policies
```

Comment:
It might be good to mention this somewhere around here:

* https://community.theforeman.org/t/organizing-access-in-foreman-seems-overly-tedious/40278/8
* https://github.com/theforeman/foreman-documentation/blob/e02c19fe9ab38bd94bde08d450134aa73c228ead/guides/common/modules/ref_predefined-roles-available-in-project.adoc?plain=1#L24